org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/typesystem/internal/AbstractTypeComputationState.java

```diff
@@ -410,15 +404,59 @@ public LightweightTypeReference getActualType(XExpression expression) {
 		Iterable<IEObjectDescription> descriptions = reentrantTypeResolver.getScopeProviderAccess().getCandidateDescriptions(
 				featureCall, XbasePackage.Literals.XABSTRACT_FEATURE_CALL__FEATURE, proxyOrResolved, featureScopeSession, demandResolvedTypes);
 		List<IFeatureLinkingCandidate> resultList = Lists.newArrayList();
+		AbstractPendingLinkingCandidate<?> previouslyResolved = resolvedTypes.forwardLinking(featureCall);
 		for(IEObjectDescription description: descriptions) {
-			resultList.add(createCandidate(featureCall, demandComputedTypes, toIdentifiableDescription(description)));
+			IFeatureLinkingCandidate candidate = createCandidate(featureCall, demandComputedTypes, toIdentifiableDescription(description));
+			// TODO apply to constructors, too
```

Comment:
Is this intentional TODO or forgotton to implement? Please double check.

Reply:
Resolved.